docker-compose.yaml

```diff
@@ -17,3 +38,8 @@ services:
       - ./nginx/localhost.key:/etc/ssl/private/localhost.key
     depends_on:
       - uploader
+    networks:
+      network:
```

Comment:
Cuando se integre los cambios de la base de datos en este PR, se debe agregar tambien el servicio de base de datos a este network. Es decir: se le debe añadir la propiedad networks a ese servicio.

Reply:
should be fixed now